integration-tests/actions/ocr2vrf_actions/ocr2vrf_steps.go

```diff
@@ -188,6 +192,53 @@ func RequestAndRedeemRandomness(t *testing.T, consumer contracts.VRFBeaconConsum
 	err = chainClient.WaitForEvents()
 	require.NoError(t, err)
 
+	requestID := getRequestId(t, consumer, receipt, confirmationDelay)
+
+	newTransmissionEvent, err := vrfBeacon.WaitForNewTransmissionEvent()
+	log.Info().Interface("NewTransmission event", newTransmissionEvent).Msg("Randomness transmitted by DON")
+
+	err = consumer.RedeemRandomness(requestID)
+	require.NoError(t, err)
```

Comment:
Make sure to name your errors when possible.

```go
require.NoError(t, err, "Error redeeming randomness from consumer")
```

Can give a lot of useful context when debugging failures.

Reply:
will add

Reply:
done